AddNodeToNodeGroup/req.validation.ts

```diff
@@ -0,0 +1,10 @@
+import * as Joi from 'joi';
+
+const reqValidation = Joi.object({
```

Comment:
```suggestion
const addNodeToNodeGroupSchema= Joi.object({
```
Change file name to `joi.ts`